health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/pages/employee/index.js

```
@@ -0,0 +1,252 @@
+      { name: "hierarchyConfig" },
+      { name: "villageRoadCondition" },
+      { name: "villageTerrain" },
+      { name: "securityQuestions" },
+      { name: "facilityType" },
+      { name: "facilityStatus" },
+      { name: "VehicleDetails" },
+      { name: "ContextPathForUser" },
+      { name: "DssKpiConfigs" },
+    ],
+    {
+      cacheTime: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)